sql/moz-fx-data-shared-prod/mozilla_org/ga_sessions/view.sql

```diff
@@ -0,0 +1,13 @@
+CREATE OR REPLACE VIEW
+  `moz-fx-data-shared-prod.mozilla_org.ga_sessions`
+AS
+SELECT
+  * REPLACE (
+    mozfun.ga.nullify_string(campaign) AS campaign,
+    mozfun.ga.nullify_string(source) AS source,
+    mozfun.ga.nullify_string(medium) AS medium,
+    mozfun.ga.nullify_string(term) AS term,
+    mozfun.ga.nullify_string(content) AS content
+  )
```

Comment:
Honestly, just because we don't have to. Because it's a 1:1 scalar transform I generally prefer to keep it as superficial as possible (just in case we want to investigate the pre-transformed data in some way).